src/main/java/fruitshop/db/Storage.java

```diff
@@ -0,0 +1,20 @@
+package fruitshop.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> storage = new HashMap<>();
+
+    public void put(String fruit, Integer quantity) {
+        storage.put(fruit, quantity);
+    }
+
+    public Integer getQuantity(String fruit) {
+        return storage.getOrDefault(fruit, 0);
+    }
+
+    public Map<String, Integer> getStorage() {
```

Comment:
The method `getStorage` exposes the internal representation of the storage. This can lead to the storage being modified in an uncontrolled manner from outside the class. It's better to return a copy or an unmodifiable view of the storage.